jp/components/grid_groupby.md

```diff
@@ -0,0 +1,140 @@
+﻿---
+title: Group By コンポーネント - ネイティブ  Angular | Ignite UI for Angular
```

Comment:
Since Group By is a feature name, not a control name, we usually translate it.  This will show up in the tab text, so it's minor.

Reply:
Translated but it appears to be a component name. e.g. "Ignite UI for Angular Group By component" and "Group By Component – Native Angular". Should component be replaced by feature?

Reply:
@yoshimis Yes, these are actually features.  They have a template for metadata and I assume the word "component" is in there.  We can leave it as-is or ask marketing why this terminology was chosen.